client/scripts/views/pages/landing/community_cards.ts

```diff
@@ -154,23 +165,23 @@ const LockdropToolsCard: m.Component<{}> = {
           },
           label: [ 'Unlock ETH ', m(Icon, { name: Icons.ARROW_RIGHT }) ],
         }),
-      ]),
+      ])
     ]);
   }
 };
 
 const NewCommunityCard: m.Component<{}> = {
   view: (vnode) => {
-    return m(Card, {
-      elevation: 1,
+    return m(FaceliftCard, {
+      elevation: 2,
       interactive: true,
-      class: 'home-card NewCommunityCard',
+      class_name: '.chain-card',
       onclick: (e) => {
         e.preventDefault();
         document.location = 'https://hicommonwealth.typeform.com/to/cRP27Rp5' as any;
       }
     }, [
-      m('.card-right', [
+      m('.lockdrop-card-body', [
```

Comment:
we don't need to name anything after the lockdrop anymore. we should use a new class here.